dist/secure-handlebars.js

```diff
@@ -10658,15 +10697,100 @@ StrictContextParser.prototype.getCurrentState = function() {
+    'action'     :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for form, input, button
+    'formaction' :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     
+    'cite'       :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for blockquote, del, ins, q
+    'poster'     :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for img, object, video, source
+    'usemap'     :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for image
+    'longdesc'   :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,                         
+    'folder'     :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for a
+    'manifest'   :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for html
+    'classid'    :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for object
+    'codebase'   :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for object, applet
+    'data'       :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for object
+    'icon'       :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for command
+    'profile'    :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for head
+    'content'    :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for meta http-equiv=refresh, kill more than need
+
+    // http://www.w3.org/TR/xmlbase/#syntax
+    'xmlns'      :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for svg, etc?
+    'xml:base'   :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE, 
+    'xmlns:xlink':StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,
+    'xlink:href' :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for xml-related
+
+    // srcdoc is the STRING type, not URI
+    'srcdoc'     :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE,     // for iframe
+
+    'style'      :StrictContextParser.ATTRIBUTE_NAME_CSS_TYPE,     // for global attributes list
+
+    // pattern matching, handling it within the function getAttributeNamesType
+    // 'on*'     :StrictContextParser.ATTRIBUTE_NAME_SCRIPTABLE_TYPE,
+
+    'type'       :StrictContextParser.ATTRIBUTE_NAME_MIME_TYPE,    // TODO: any potential attack of the MIME type?
+
+    'rel'        :{'link'    :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE},
+    'value'      :{'param'   :StrictContextParser.ATTRIBUTE_NAME_URI_TYPE},
+};
 
+/**
+ * @function StrictContextParser#getAttributeNamesType
+ *
+ * @returns {integer} the attribute type defined for different handling
+ *
+ * @description
+ * Check if the current tag can possibly incur script either through configuring its attribute name or inner HTML
+ *
+ */
+StrictContextParser.prototype.getAttributeNamesType = function() {
+    if (this.attributeName.match(/^on/i)) { /* assuming it is from Strict Context Parser.
+                                               and o{{placeholder}}n* can bypass the check.
+                                               anyway, we are good to throw error in atttribute name state. */
+        return StrictContextParser.ATTRIBUTE_NAME_SCRIPTABLE_TYPE;
```

Comment:
just my two cents, calling it `ATTRTYPE_EVENT`might be sufficient. the choice of EVENT sounds closer to the spec